src/test.ml

```diff
@@ -358,6 +359,72 @@ let test_agg_nullable = [
   |} [ attr' "result" Int; ][];
 ]
 
+let test_ambiguous = [
+  tt "CREATE TABLE test21 (id INT, column_a TEXT, column_b BOOL)" [] [];
+  tt "CREATE TABLE test22 (id INT, column_d INT)" [] [];
+  wrong "select id from test21 join test22 on test21.id = test22.id order by id";
+  tt "select test21.id from test21 join test22 on test21.id = test22.id order by id" [
+    attr' ~nullability:(Nullable) "id" Int;
+  ] [];
+  wrong "select test21.id from test21 join test22 on test21.id = test22.id where id > 2 order by id";
+  tt "select test21.id from test21 join test22 on test21.id = test22.id group by id" [
+    attr' ~nullability:(Nullable) "id" Int;
+  ] [];
+  tt "select test21.id as test from test21 join test22 on test21.id = test22.id group by column_a" [
+    attr' ~nullability:(Nullable) "test" Int;
+  ][];
+  tt "select test21.id, test22.id from test21 join test22 on test21.id = test22.id" [
+    attr' ~nullability:(Nullable) "id" Int;
+    attr' ~nullability:(Nullable) "id" Int;
+  ] [];
+  wrong "select id, id from test21 join test22 on test21.id = test22.id group by id";
+  wrong "select test21.id, test22.id from test21 join test22 on test21.id = test22.id group by id";
+  tt "select test21.id from test21 join test22 on test21.id = test22.id group by id, column_a" [
+    attr' ~nullability:(Nullable) "id" Int;
+  ] [];
+  tt "SELECT COUNT(column_a) as column_a FROM test21 WHERE column_a = @column_a" [
+    attr' "column_a" Int;
+  ] [
+    named "column_a" Int;
+  ];
+  wrong "select * from test21 join test22 on test21.id = test22.id group by id" ;
+  tt "CREATE TABLE test23 (id INT)" [] [];
+  tt "CREATE TABLE test24 (id INT)" [] [];
+  wrong "select * from foo join bar on foo.id = bar.id order by id";
+  wrong "select * from foo join bar on foo.id";
+  tt "SELECT test21.id AS id, test22.id AS id FROM test21 JOIN test22 ON test21.id = test22.id" [
```

Comment:
```suggestion
  tt "SELECT test21.id AS id1, test22.id AS id2 FROM test21 JOIN test22 ON test21.id = test22.id" [
```
nobody should write two AS id so lets not confuse ppl

Reply:
Fixed